react-native/react/actions/config/index.js

```diff
@@ -19,20 +19,27 @@ export function startup () {
         dispatch({type: Constants.startupLoaded, payload: error, error: true})
         return
       }
-      engine.rpc('config.getCurrentStatus', {}, incomingMap, (error, status) => {
-        if (error) {
-          dispatch({type: Constants.startupLoaded, payload: error, error: true})
-          return
-        }
-        dispatch({
-          type: Constants.startupLoaded,
-          payload: {config, status}
+
+      function getCurrentStatus () {
```

Comment:
doing this would require more code than is worth it right now so :+1: for this. What could happen is

- Make both rpc calls one after the other (don't serialize them)
- When config comes back, inject it into the store, when status comes back inject it into the store
- The reducer of both will call a shared function which checks to see if both are valid and will do what the current startupLoaded does. 
- Autologin could likely be left in the getStatus call, or we could parameterize the action and not do it on reconnect